CHANGES.md

```diff
@@ -1,6 +1,9 @@
 unreleased
 -------------------
 
+- Sort embedded errors that are appended to the AST by location so the compiler
+  reports the one closer to the beginning of the file first.
```

Comment:
Could you add your GH handle and the PR number?

Reply:
Sure!